doc/source/flexibility/proxy_protocol_support.rst

```diff
@@ -4,17 +4,16 @@
  Support for PROXY protocol
 ============================
 
-The proxy protocol allows an intermediate proxying server speaking proxy protocol (ie. HAProxy) between the server and the ultimate client (i.e. mysql client etc) to provide the source client address to the server, which normally would only see the proxying server address instead.
 
-As the proxy protocol amounts to spoofing the client address, it is disabled by default, and can be enabled on per-host or per-network basis for the trusted source addresses where trusted proxy servers are known to run. Unproxied connections are not allowed from these source addresses.
+The proxy server provides the benefits of load balancing, and security. Proxy protocol adds a header with connection information to the client's connection to the destination. Use the proxy protocol to pass the client IP address to a destination, which, normally, would only see the proxy server's address.
 
-.. note:: 
+You can enable the protocol on a per-host or a per-network basis for the trusted source addresses, where trusted proxy servers are known. You should specify from which network the destination expects the proxy headers. The `proxy_protocol_networks` should be set to the proxy address or a dedicated network range. The protocol is supported for TCP over IPv4 and IPv6 connections only. UNIX socket connections can not use the proxy protocol and do not fall under the effect of `proxy-protocol-networks`='*'. As a special exception, it is forbidden for the proxy protocol IP address to be ``127.0.0.1`` or ``::1``.
 
-   You need to ensure proper firewall ACL's in place when this feature is enabled. 
+Connections that do not use the proxy server are not allowed from the proxy-dedicated range. When the `proxy_protocol_networks` is enabled, the client communicates first. When a connection does not use the protocol, the server communicates first with an Initial Handshake packet and the client responds. If `proxy_protocol_networks` is enabled, a dedicated client address cannot connect to the destination without the proxy header. This connection stalls with no response and no error.
 
-Proxying is supported for TCP over IPv4 and IPv6 connections only. UNIX socket connections can not be proxied and do not fall under the effect of proxy-protocol-networks='*'.
+.. note:: 
 
-As a special exception, it is forbidden for the proxied IP address to be ``127.0.0.1`` or ``::1``.
+   You need to ensure proper firewall Access Control Lists in place when this feature is enabled. 
```

Comment:
I would change it to ->  Ensure proper firewall Access Control Lists are in place when this feature is enabled.